.github/workflows/ci.yml

```diff
@@ -47,8 +47,18 @@ jobs:
         with:
           version: nightly
 
-      - name: Build
-        run: forge build
+      - name: Anvil1
+        run: anvil &
+      - name: Anvil2
+        run: anvil --chain-id 1234 -p 8555 &
+
+      - name: Deploy core
+        run: |
+          DEBUG=hyperlane* yarn ts-node scripts/deploy.ts --local anvil1 --remotes anvil2 --no-write-agent-config --key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
I guess these keys are probably not valuable but would still recommend moving them to a github CI env var instead of hardcoding.

Reply:
They're just the default anvil keys